build/angular-date-range-picker.js

```diff
@@ -173,6 +173,10 @@
                 $event.stopPropagation();
               }
             }
+            if ($scope.selecting) {
+              $scope.selection = moment().range($scope.start, $scope.start.clone().endOf("day"));
```

Comment:
Here is a bug.

STR:
1. Open selector, choose any date and click "Cancel"
2. Open selector again and click "Apply". In console we'll see: "TypeError: Cannot read property 'clone' of null", window won't be closed.
3. Select any date, click "Apply"

Expected result:
Value should be set to clicked date.

Actual result:
The same as it was before: Invalid date, calendar is broken etc